services/sourceforge/sourceforge-tickets.service.js

```diff
@@ -0,0 +1,65 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { metric } = require('../text-formatters')
+const { BaseJsonService } = require('..')
+
+const schema = Joi.object({
+  count: Joi.number().required(),
+}).required()
+
+module.exports = class SourceforgeTickets extends BaseJsonService {
+  static get category() {
+    return 'other'
+  }
+
+  static get route() {
+    return {
+      base: 'sourceforge/tickets',
+      pattern: ':type(bugs|feature-requests)/:project',
```

Comment:
I feel like `project` should probably come before the type
```suggestion
      pattern: ':project/:type(bugs|feature-requests)',
```

Reply:
I did it this way as that was how it was done in the base sourceforge badge:
`:interval(dt|dm|dw|dd)/:project/:folder`

But if you'd rather it be before the type then that's fine :)

Reply:
So the difference there is that the interval represents the noun portion of that route. https://github.com/badges/shields/blob/master/CONTRIBUTING.md#badge-urls

The badge for that service is for downloads, whereas the badge here is for open ticket counts, so the `interval` is the noun there, and the noun here is `tickets`.  Both `project` and `folder` in the SourceForge downloads route and `project` and `type` here in this service are params/qualifiers. I personally think having `project` as the first "param" (in the context of the full badge url) makes more sense